installation/scripts/generate-local-config.ps1

```diff
@@ -24,6 +26,18 @@ Invoke-Expression -Command $cmd
 
 ##########
 
+Write-Output "Configuring versions ..."
+
+if([System.IO.File]::Exists($VERSIONS_FILE_PATH)){
+    $cmd = "kubectl create configmap versions --from-env-file=${VERSIONS_FILE_PATH} -n `"kyma-installer`""

```

Comment:
Please make the output of PS script similar to sh

Reply:
in sh shouldn't be that echo instead